docs/setup/configuration.mdx

```diff
@@ -236,3 +240,24 @@ Saleor can automatically convert amounts between currencies. To use this feature
 ### `OPENEXCHANGERATES_API_KEY`
 
 Your store’s Open Exchange Rates "App ID".
+
+## Updating domain configuration
+
+When Saleor is exposed at a domain (e.g. `api.example.com`), it must be configured to generate URLs properly. Follow these steps to update the domain configuration:
+
+1. Use the GraphQL `shopDomainUpdate` mutation shown below to update Saleor's shop domain configuration with the new one.
+   ```graphql
+   mutation UpdateDomain {
+     shopDomainUpdate(input: { domain: "api.example.com" }) {
+       shop {
+         domain {
+           host
+         }
+       }
+     }
+   }
+   ```
+   Note that Saleor instances must be restarted for the new configuration to take effect, as shop data is cached in memory.
```

Comment:
I don't believe that's true.

Reply:
From what I've checked, the `Site` objects are stored in memory, and whenever `Site.objects.get_current()` is called, the object is retrieved from the cache if available. However, if Saleor is operating on multiple processes/workers, this can result in `Site.objects.get_current()` returning outdated information (for example, for URL generation). I suggested this "restarting" solution, but I could be missing some crucial details.

Reply:
Saleor used to disable the cache. If the cache is still in effect, please file a critical bug. There is no way to restart Saleor in a clustered cloud environment.

Reply:
https://github.com/saleor/saleor/issues/12528